### DISCUSSION TITLE
API App (App Service) i Linux App Service Plan

### DISCUSSION THREAD

**Discussion by maciejgos** [AzureCommunityPL/group#86] [Pomoc] [answered]
Cześć próbuje odpalić API na planie Linux głównie ze względu na koszty, ale niestety przy próbie tworzenie aplikacji nie widzę w ogóle utworzonego planu chodzącego na Linux.

Wszystkie zasoby są w ramach tej samie RG i w tym samym regionie. Jak na złość nie mogę nic znaleźć w dokumentacji z takim problemem.

Może ktoś coś podpowiedzieć czemu jest takie zachowanie, skoro API App i Web APP pod spodem to jest to samo? Chyba, że się mylę 😉.

PS. Dla potomnych, jeżeli wybierzemy Web App to mamy dostęp do planu Linux 🤣

**Comment by kamil-mrzyglod** [comment, marked as answer]
Ja tam od dawna tej "podusługi usługi" nie śledzę, ale czy przypadkiem docelowo nie miała być zastąpiona integracją z APIM? Jakiej dokładnie funkcjonalności szukasz, że chcesz wykorzystać API App a nie generyczny App Service? W CLI nie widzę takiej separacji i na ten moment jedyne co kojarzę to `kind` w szablonach ARM gdzie mogłeś faktycznie wskazać na API App.

W portalu da się stworzyć API App(czy też App Service w wersji `api`), ale to, że nie możesz wybrać planu z Linux nie musi wynikać, że coś nie jest wspierane tylko akurat UI już z lekka trąci myszką 😛

**Reply by maciejgos** [reply]
@kamil-mrzyglod to chyba jest faktycznie ten case, że UI nie nadąża 😉. Zarówno API App jak i Web App kierują do tej samej dokumentacji, czyli powinno być tak samo.

Póki co zabawę z planem linux zakończyłem bo jak się okazało jest również problem z Application Insights.
